lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -1054,6 +1054,34 @@ static LogicalResult verifyStdIntCast(StdIntCast cast) {
   return success();
 }
 
+//===----------------------------------------------------------------------------===//
+// Conversions from flipped fixed-width signless integer types in standard
+// dialect.
+//===----------------------------------------------------------------------------===//
+
+static LogicalResult verifyStdFlippedIntCast(StdFlippedIntCast cast) {
+  // Either the input or result must have signless standard integer type, the
+  // other must be a FIRRTL type that lowers to one, and their widths must
+  // match.
+  FIRRTLType firType;
+  FIRRTLType passiveType;
+  if (firType = cast.getOperand().getType().dyn_cast<FlipType>()) {
+    passiveType = firType.getPassiveType();
+  } else {
+    cast.emitError("firrtl type is not flipped");
+    return failure();
+  }
+
+  int32_t intWidth = passiveType.getBitWidthOrSentinel();
+  if (intWidth == -2)
+    return cast.emitError("flipped firrtl type isn't simple bit type");
```

Comment:
Does this need a ", failure()"?  If not, you can probably drop it from the next line